editor/src/components/editor/insertmenu.tsx

```diff
@@ -106,43 +99,15 @@ interface InsertMenuProps {
 export const InsertMenu = betterReactMemo('InsertMenu', () => {
   const props = useEditorState((store) => {
     const openFileFullPath = getOpenFilename(store.editor)
-    let currentlyOpenFilename: string | null = null
-    if (openFileFullPath != null) {
-      const splitFilename = openFileFullPath.split('/')
-      currentlyOpenFilename = defaultIfNull<string | null>(null, last(splitFilename))
-    }
-
-    let currentFileComponents: Array<CurrentFileComponent> = []
     const openUIJSFile = getOpenUIJSFile(store.editor)
-    if (openUIJSFile != null && openFileFullPath != null) {
-      forEachParseSuccess((fileContents) => {
-        Utils.fastForEach(fileContents.topLevelElements, (topLevelElement) => {
-          if (isUtopiaJSXComponent(topLevelElement)) {
-            const componentName = topLevelElement.name
-            const defaultProps = defaultPropertiesForComponentInFile(
-              componentName,
-              dropFileExtension(openFileFullPath),
-              store.editor.propertyControlsInfo,
-            )
-            const detectedProps = topLevelElement.propsUsed
-            currentFileComponents.push({
-              componentName: componentName,
-              defaultProps: defaultProps,
-              detectedProps: detectedProps,
-            })
-          }
-        })
-      }, openUIJSFile.fileContents.parsed)
-    }
```

Comment:
Don't delete this code please. I hadn't realised we had regressed and lost the default and detected props used for insertion in #969. Looks like it shouldn't be too tricky to restore that.

Reply:
I'll put a comment on that issue specifically mentioning this code snippet, because the change I made for this fix affects this code as well so I'll be modifying dead code that is never (currently) being run or tested. Then whenever someone works on that issue they can see what was there at one point.

Reply:
Yep ok works for me